scripts/devaccess/devaccess_wrap.py

```diff
@@ -270,19 +304,51 @@ def rpm_qa(cmd):
         return results
 
     @staticmethod
-    def oc_get_nodes(occmd):
-        ''' provide 'oc get nodes' '''
+    def cmd_with_tail(cmd):
+        ''' replace process with called one to keep getting updates '''
+        args = cmd.split()
+        os.execvp(args[0], args)
 
-        n_cmd = occmd.normalized_cmd()
-        run_cmd = WhitelistedCommands.oc_cmd_builder(n_cmd)
-        results = subprocess.check_output(run_cmd)
+    @staticmethod
+    def safe_command(cmd):
+        ''' return any safe command output
+            this is for all commands that do not have any type of sensitive data in them
+        '''
+        results = ''
+        our_env = os.environ
+        our_env['PATH'] = '/usr/sbin:' + our_env['PATH']
+        try:
+            results = subprocess.check_output(cmd.split(), env=our_env)
+        except subprocess.CalledProcessError:
+            log.exception("Call to process (%s) failed", cmd)
 
         return results
 
     @staticmethod
-    def oc_get_pods(occmd):
-        ''' provide 'oc get pods' '''
+    def super_safe_command(cmd):
```

Comment:
Suggest renaming as something like "sudo_safe_command", as "super safe" is too easily confused for the descriptive "very safe".